### PR TITLE
CVSL-376 prod helm values

### PR DESCRIPTION
**Pull request by timharrison-moj**
* CircleCI build pipeline to promote to prod.
* `values-prod.yaml` config

### CODE REVIEW

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    PRISONER_SEARCH_API_URL: "https://prisoner-offender-search.prison.service.justice.gov.uk"
+    PROBATION_SEARCH_API_URL: "https://probation-offender-search.hmpps.service.justice.gov.uk"
+    PRISON_REGISTER_API_URL: "https://prison-register.hmpps.service.justice.gov.uk"
+    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    PHASE_BANNER_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    LICENCE_WATERMARK: "false"
+    # A reference to the Gotenberg service - to submit HTML to PDF conversions (internal within namespace)
+    GOTENBERG_API_URL: "http://create-and-vary-a-licence-gotenberg"
+    # A reference to the CVL service - for links in HTML to get images/stylesheets (internal within namespace)
+    LICENCES_URL: "http://create-and-vary-a-licence"
```
Comment (smcveigh941) [inline helm_deploy/values-prod.yaml:27]: You need the ENV variable to restrict rollout areas. Although if we want to test on the prod environment then leave it for now!

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  env:
+    GOOGLE_CHROME_IGNORE_CERTIFICATE_ERRORS: 1
+    DISABLE_UNOCONV: 1
+    DEFAULT_WAIT_TIMEOUT: 30
+
+delius-wiremock:
+  replicaCount: 0
+  env:
+    HMPPS_AUTH_URL: "https://sign-in.hmpps.service.justice.gov.uk/auth"
+    PRISONER_SEARCH_API_URL: "https://prisoner-offender-search.prison.service.justice.gov.uk"
```
Comment (smcveigh941) [inline helm_deploy/values-prod.yaml:40]: These wouldnt be needed since delius-wiremock wont exist. Maybe its broken without them not sure

**Review by smcveigh941 — APPROVED** [review]
LGTM

File: helm_deploy/values-prod.yaml (hunk 2 above)
Comment (timharrison-moj) [reply helm_deploy/values-prod.yaml:40]: I think they need to exist, but with `replicaCount = 0` no pods get created.

File: helm_deploy/values-prod.yaml (hunk 1 above)
Comment (timharrison-moj) [reply helm_deploy/values-prod.yaml:27]: Yes, left this out for now whilst we check it. Only one environment variable required `RESTRICT_ROLLOUT=true` and would then read the probation areas and prisons from `config.js`.

**Pull request merged**